pinot-query-planner/src/test/resources/queries/LiteralEvaluationPlans.json

```diff
@@ -160,6 +160,29 @@
           "\n"
         ]
       },
+      {
+        "description": "select two functions chaining",
+        "sql": "EXPLAIN PLAN FOR SELECT ToEpochDays(fromDateTime('1970-01-15', 'yyyy-MM-dd')) FROM a",
+        "output": [
+          "Execution Plan",
+          "\nLogicalProject(EXPR$0=[14:BIGINT])",
+          "\n  LogicalTableScan(table=[[a]])",
+          "\n"
+        ]
+      },
+      {
+        "description": "filter with two functions chaining",
+        "sql": "EXPLAIN PLAN FOR SELECT count(*) FROM a WHERE col1 > ToEpochDays(fromDateTime('1970-01-15', 'yyyy-MM-dd'))",
+        "output": [
+          "Execution Plan",
+          "\nLogicalAggregate(group=[{}], agg#0=[COUNT($0)])",
+          "\n  PinotLogicalExchange(distribution=[hash])",
+          "\n    LogicalAggregate(group=[{}], agg#0=[COUNT()])",
+          "\n      LogicalFilter(condition=[>(CAST($0):BIGINT NOT NULL, 14)])",
```

Comment:
could you add another test case where CAST is not being involved (to validate the other codepath)

Reply:
done.